public/app/percona/add-instance/components/AddRemoteInstance/FormParts/MainDetails/MainDetails.tsx

```diff
@@ -7,11 +7,13 @@ import { MainDetailsFormPartProps } from '../FormParts.types';
 import { getStyles } from '../FormParts.styles';
 import { Messages } from '../FormParts.messages';
 
-export const MainDetailsFormPart: FC<MainDetailsFormPartProps> = ({ remoteInstanceCredentials }) => {
+export const MainDetailsFormPart: FC<MainDetailsFormPartProps> = ({ form, remoteInstanceCredentials }) => {
   const theme = useTheme();
   const styles = getStyles(theme);
+  const tlsFlag = form && form.getState().values && form.getState().values['tls'];
```

Comment:
can we just save `form.getState().values` before, to avoid calling it twice?

Reply:
Done!